alvr/client_core/src/graphics/lobby.rs

```diff
@@ -222,7 +222,7 @@ impl LobbyRenderer {
             device,
             "lobby_line",
             &[],
-            64,
+            68,
```

Comment:
This should get unhardcoded in some way or at least commented and extracted into a constant (also goes for all the 64s.